v4l2/v4l2_encode.cpp

```diff
@@ -297,6 +298,10 @@ int32_t V4l2Encoder::ioctl(int command, void* arg)
             switch (format->fmt.pix_mp.pixelformat) {
                 case V4L2_PIX_FMT_H264: {
                     m_encoder.reset(createVideoEncoder(YAMI_MIME_H264), releaseVideoEncoder);
+                    if (not m_encoder) {
```

Comment:
Ah yes, I'll follow the project convention... I don't have a problem with that.  I'll resubmit with that change.